packages/core/src/utilities/worldToImageCoords.ts

```diff
@@ -59,10 +63,11 @@ function worldToImageCoords(
   ];
 
   if (
-    imageCoords[0] < 0 ||
-    imageCoords[0] >= columns ||
-    imageCoords[1] < 0 ||
-    imageCoords[1] >= rows
+    checkCoordinateSpace &&
```

Comment:
Only check coordinates if requested to do so - normally that isn't a useful thing to do inside the method, as there are quite a few cases where at least one coordinate can usefully be outside by nearby the image.